### PR TITLE
btrfs-progs: new package at 5.4.1

### PR DESCRIPTION
**Pull request by aicsx**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/btrfs-progs/version
/var/db/kiss/installed/btrfs-progs/sources
/var/db/kiss/installed/btrfs-progs/manifest
/var/db/kiss/installed/btrfs-progs/depends
/var/db/kiss/installed/btrfs-progs/checksums
/var/db/kiss/installed/btrfs-progs/build
/var/db/kiss/installed/btrfs-progs/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/udev/rules.d/64-btrfs-dm.rules
/usr/lib/udev/rules.d/
/usr/lib/udev/
/usr/lib/libbtrfsutil.so.1.1.1
/usr/lib/libbtrfsutil.so.1
/usr/lib/libbtrfsutil.so
/usr/lib/libbtrfsutil.a
/usr/lib/libbtrfs.so.0.1
/usr/lib/libbtrfs.so.0
/usr/lib/libbtrfs.so
/usr/lib/libbtrfs.a
/usr/lib/
/usr/include/btrfsutil.h
/usr/include/btrfs/version.h
/usr/include/btrfs/sizes.h
/usr/include/btrfs/send.h
/usr/include/btrfs/send-utils.h
/usr/include/btrfs/send-stream.h
/usr/include/btrfs/rbtree.h
/usr/include/btrfs/raid56.h
/usr/include/btrfs/radix-tree.h
/usr/include/btrfs/list.h
/usr/include/btrfs/kerncompat.h
/usr/include/btrfs/ioctl.h
/usr/include/btrfs/extent_io.h
/usr/include/btrfs/extent-cache.h
/usr/include/btrfs/ctree.h
/usr/include/btrfs/crc32c.h
/usr/include/btrfs/btrfsck.h
/usr/include/btrfs/btrfs-list.h
/usr/include/btrfs/
/usr/include/
/usr/bin/mkfs.btrfs
/usr/bin/fsck.btrfs
/usr/bin/btrfstune
/usr/bin/btrfsck
/usr/bin/btrfs-select-super
/usr/bin/btrfs-map-logical
/usr/bin/btrfs-image
/usr/bin/btrfs-find-root
/usr/bin/btrfs-convert
/usr/bin/btrfs
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

**Comment by jedavies-dev** [comment]
Add the manifest.

**Comment by aicsx** [comment]
yes, i add it. Thank you.

**Comment by dylanaraps** [comment]
See this link that @jedavies-dev posted: https://github.com/kisslinux/community/commit/fe314f29e9252c42d230fc1aa0535c124a3628eb

File: community/btrfs-progs/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr \
+            --bindir=/bin \
+            --libdir=/lib \
```
Comment (dylanaraps) [inline community/btrfs-progs/build]:
```sh
./configure \
    --prefix=/usr \
    # the rest here
```

File: community/btrfs-progs/build [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-            --disable-documentation \
-            --disable-backtrace \
-            --disable-zstd
+./configure \
+	--prefix=/usr \
+	--bindir=/bin \
+	--libdir=/lib \
+	--disable-documentation \
+	--disable-backtrace \
+	--disable-zstd
```
Comment (aicsx) [inline community/btrfs-progs/build]: i think it's ok now.

File: community/btrfs-progs/build (hunk 1 above)
Comment (aicsx) [reply community/btrfs-progs/build]: build changes apply. View, now please.

File: community/btrfs-progs/build [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+./configure \
+	--prefix=/usr \
+	--bindir=/bin \
+	--libdir=/lib \
```
Comment (dylanaraps) [inline community/btrfs-progs/build]: Remove these two and post the updated manifest.

**Comment by dylanaraps** [comment]
Looks like we need `--bindir=/usr/bin`.

**Pull request merged**